packages/core/src/vuePlugins/vuex.ts

```diff
@@ -141,6 +141,12 @@ export const makeStore = () => {
         noop(state.selected)
         return selected
       },
+      selectedCoordinates: (state) => {
+        noop(state.selected)
+        return selected === null
+          ? null
+          : (selected.getGeometry() as Point).getCoordinates()
+      },
```

Comment:
This addition was required for the GFI to react to the selected feature directly. In the Meldemichel, the reaction is solved differently via a `dispatchOnMapSelect` that opens the GFI as part of the menu, which is not the way for this plugin in general, since the GFI may be not part of the IconMenu, as it is in the Snowbox.